test/com/gfredericks/test/chuck/clojure_test_test.cljc

```diff
@@ -12,9 +12,16 @@
     (is (< i 0))))
 
 (deftest options-test
+  ;; no option is OK, defaults to 100 tests
+  (let [nb-runs (atom 0)]
+    (checking "no option works" [i gen/s-pos-int]
+              (swap! nb-runs inc)
+              (is (> 0)))
+    (testing "no option means 100 runs (test.check's default)"
+      (is (= 100 @nb-runs))))
   ;; empty map is OK, defaults to 100 tests
   (checking "strings are strings" {} [s gen/string-ascii]
-    (is (string? s)))
+            (is (string? s)))
```

Comment:
try not to change existing indentation; I'd also prefer the `checking` call farther up to be indented 2 spaces as well